Jenkinsfile.groovy

```diff
@@ -1,5 +1,5 @@
 def commit = "UNKNOWN"
-
+def githubContainerRepo = "ghcr.io/overture-stack" 
```

Comment:
I've personally named it dockerRepo as technically the pattern is the same regardless of registry.

Reply:
true, I just differentiated it because we still pushing to docker hub here too

Reply:
renamed